examples/with-remix/remix.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   serverDependenciesToBundle: [
     '@rainbow-me/rainbowkit',
     '@rainbow-me/rainbowkit/wallets',
-    'wagmi',
-    '@wagmi/core',
+    /^@?wagmi.*/,
+    'zustand/vanilla',
```

Comment:
@ertyurk what's reasoning for adding zustand here as well? I'm unfamiliar with some of the remix config.

Reply:
Hi. afaik, zustand/vanilla is used at wagmi as well. without it, the Remix JS example did not work at that time.